app/models/miq_report/seeding.rb

```diff
@@ -62,8 +62,8 @@ def sync_from_file(filename, dir, typ)
       rec = find_by_filename(rpt[:filename])
 
       if rec
-        if rec.filename && (rec.file_mtime.nil? || rec.file_mtime.utc < rpt[:file_mtime])
-          _log.info("#{typ.titleize}: [#{rec.name}] file has been updated on disk, synchronizing with model")
+        if rec.filename
+          _log.info("#{typ.titleize}: [#{rec.name}] file exists, synchronizing with model")
```

Comment:
Ok, so I'm fine with these changes but I don't know if anything will be changed when we're constantly touching the report objects.  For example, do we have report definitions sorted in the UI by updated on?

I don't think so but it might be worth running this through several server boots and verifying the report pages show the reports in the same order in some screens.

Reply:
sorting does changes slightly between server reboots

Reply:
i think changing sorting is result of change, since before change reports record on DB was not touched.
Note: there is different sorting on left part (in tree) and on right part (in table) 

before: (on master):
![screen shot 2017-09-29 at 9 23 42 am](https://user-images.githubusercontent.com/6556758/31018389-e1571c00-a4f9-11e7-877b-63c6e11c67b8.png)



after (on this branch):
reboot 1  (red marks difference between rebootes):
<img width="601" alt="branch1" src="https://user-images.githubusercontent.com/6556758/31018345-b82455be-a4f9-11e7-9f2d-0d91c2a2e4a2.png">

reboot 2)
![branch2](https://user-images.githubusercontent.com/6556758/31018371-d6dcbfb4-a4f9-11e7-8411-9c6619a2cff0.png)

Reply:
@yrudman can you find how they're sorting these?

Reply:
@jrafanie I think there is no explicit sorting, also it does look inconsistent now when on the same screen there are same lists side by side, and those list have different order.
I think it make sense to sort those list of reports in the same way alphabetically.
Are you thinking of changing sorting in this PR ?

Reply:
I am still debugging to find-out how exactly sorting done on UI side. At this point I can say how right side sorted - it is sorted by yaml file name (`SELECT "miq_reports".* FROM "miq_reports"  ORDER BY "miq_reports"."rpt_type" ASC, "miq_reports"."filename" ASC, "miq_reports"."name" ASC`) and report with filename `005_VMs with Free Space _ 50% by Department` will be first (which is shown in right side of screenshots.

It looks like the right side in report tree formed by using this SQL: `SELECT "miq_reports".* FROM "miq_reports" WHERE "miq_reports"."name" IN ('VMs with Free Space > 50% by Department', 'VMs w/Free Space > 75% by Function', 'VMs w/Free Space > 75% by LOB', 'VM Disk Usage', 'Hardware Information for VMs', 'Vendor and Type', 'Vendor and Guest OS', 'VM Location and Size', 'VM UUIDs', 'VMs with no UUID', 'VMs with Volume Free Space <= 20%', 'VMs with Volume Free Space >= 80%', 'VMs by MAC Address', 'Unregistered VMs', 'Orphaned VMs', 'VMs Snapshot Summary', 'User Accounts - Windows', 'User Accounts - Linux', 'Account Groups - Windows', 'Account Groups - Linux', 'Guest OS Information - any OS', 'Guest OS Information - Windows', 'Guest OS Information - Linux', 'Guest OS Password Information - Windows', 'Guest OS HKLM Registry Information')`  
and this SQL do not have  any explicit sorting.  I did not find (yet) any sorting on UI side

Reply:
@yrudman Can you confirm the order of these reports change on master without your changes?  Based on your findings of the sql queries, I'd guess this would happen without your changes.

Reply:
not sure, i've tried couple time and master still the same. I think the order without explicit ordering is not guaranteed, but since data was not changed with reboot it is the same, will try more on master ...

Reply:
@jrafanie I was wrong in above comment - sorting slightly changing on master too